addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2272,10 +2272,6 @@ var SnippetsMenu = Widget.extend({
      * @returns {Promise}
      */
     callPostSnippetDrop: async function ($target) {
-        this.postSnippetDropPromise = new Promise(resolve => {
-            this._postSnippetDropResolver = resolve;
-        });
```

Comment:
Let's make this stable: this should stay here. To do the same thing in a stable way, you can put the synchronous code that is after the call to `callPostSnippetDrop` inside a function and call that function before the call to `_postSnippetDropResolver`. To make that stable, something like that:


```javascript
self.__postSnippetDropExtraActions = () => {
    // Restore editor to its normal edition state, ... blablabla
};
await self.callPostSnippetDrop($target);
```
```javascript
callPostSnippetDrop: async function ($target) {
    this.postSnippetDropPromise = new Promise(resolve => {
        this._postSnippetDropResolver = resolve;
    });
    // [...]
    if (this.__postSnippetDropExtraActions) {
        this.__postSnippetDropExtraActions();
        delete this.__postSnippetDropExtraActions;
    }
    this._postSnippetDropResolver();
```

Then hopefully, in master, we'll make something clean.